frontend/src/components/PageNotFoundComponent.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="container">
+    <v-img src="../assets/404logo.png" id="err-logo"></v-img>
+    <v-img src="../assets/404img.png" alt="logo" id="err-img"></v-img>
+    <div class="row">
+      <v-btn
+        type="ReturnToHome"
+        block
+        class="mt-2 btn"
+        @click="$router.push('/')"
+        >Return To Home</v-btn
+      >
+      <v-btn
+        type="ReturnToPrevious"
```

Comment:
the button type should be `text`, `number`, etc.. not `ReturnToPrevious, Please update